sql/2019/07_Performance/07_14.sql

```diff
@@ -0,0 +1,12 @@
+#standardSQL

```

Comment:
A few minor suggestions:

```sql
#standardSQL
# 07_14: Percentiles of visually complete metric
SELECT
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(100)] / 1000, 2) AS p10,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(250)] / 1000, 2) AS p25,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(500)] / 1000, 2) AS p50,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(750)] / 1000, 2) AS p75,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(900)] / 1000, 2) AS p90
FROM
  `httparchive.summary_pages.2019_07_01_*`
```

- tweaked indentation and whitespace
- no need for n+1 quantiles/offset, you can use round values like 1000 and 500 (this was my fault when I didn't realize the functions already did n+1, so many example queries are off by one)
- use backticks around the table name
- when testing queries locally you can query the 1k tables for efficiency but in PRs you should reference the full dataset (httparchive.summary_pages.2019_07_01_*)
- use 2019_07_01_* to capture both desktop and mobile

You may also want to group by desktop/mobile:

```sql
#standardSQL
# 07_14: Percentiles of visually complete metric
SELECT
  _TABLE_SUFFIX AS client,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(100)] / 1000, 2) AS p10,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(250)] / 1000, 2) AS p25,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(500)] / 1000, 2) AS p50,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(750)] / 1000, 2) AS p75,
  ROUND(APPROX_QUANTILES(visualComplete, 1000)[OFFSET(900)] / 1000, 2) AS p90
FROM
  `httparchive.summary_pages.2019_07_01_*`
GROUP BY
  client
```

Reply:
@zeman any preference for splitting out desktop/mobile or seeing everything in one distribution?

Reply:
@rviscomi if we can get a split on mobile vs desktop that would be great.

Reply:
@rviscomi  thanks for your feedback. Not very well versed with the next steps - do I close this PR and  make the changes in my branch? and then merge it with the main master branch later?

Reply:
Keep adding to this PR with all of your metrics for this chapter and when they've all been reviewed we'll merge the PR.